apps/session-manager/src/services/__tests__/redis-session-storage.spec.ts

```diff
@@ -214,19 +220,15 @@ describe("RedisSessionStorage#getBySessionToken", () => {
   });
 
   test("should fail parse of user payload", async () => {
-    mockGet.mockImplementationOnce((_) => Promise.resolve("Invalid JSON"));
+    mockGet.mockImplementationOnce((_) => Promise.resolve(INVALID_JSON));
```

Comment:
a little bit more compact
```suggestion
    mockGet.mockResolvedValueOnce(INVALID_JSON);
```

Reply:
Done in [[#IOPID-1831] refactor based on comments](https://github.com/pagopa/io-auth-n-identity-domain/pull/21/commits/f7172071277aa811bf0b9d7c392ed059f7a54b29)